package.json

```diff
@@ -32,6 +32,7 @@
   "devDependencies": {
     "@typechain/ethers-v5": "^1.0.0",
     "@types/chai": "^4.2.11",
+    "@types/expect": "^24.3.0",
```

Comment:
what's this?

Reply:
Actually I can remove it

Reply:
Leftovers from attempt to use `ts-mocha`